### PR TITLE
Configurable volume attributes

### PR DESCRIPTION
**Pull request by spriya-m**
# Description
Documentation updates for Configurable volume attributes feature

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| dell/csm#291 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/features/powerstore.md [hunk 1]
```diff
@@ -681,3 +681,67 @@ In case blockProtocol is specified as `auto`, the driver will be able to find th
...
+          storage: 8Gi
+  storageClassName: powerstore-ext4
+
+```
+
+>Note: Default description value is <pvc-name>-<pvc-namespace>. 
+
+The following is the list of all the attribtues supported by PowerStore CSI driver: 
+
+1. Block Volume Attributes:
```
[the remark this answers is not in the document]
Comment (spriya-m) [reply content/docs/csidriver/features/powerstore.md]: Done

**Comment by shanmydell** [comment]
image

When given in <> the values will not be printed in md files , please have a look

**Comment by shanmydell** [comment]
Release notes - Updates please @spriya-m

**Review by harshitap26 — DISMISSED** [review]

**Comment by spriya-m** [comment]
@shanmydell , I have fixed the issue with content within <> and have updated the release notes.

**Review by shanmydell — APPROVED** [review]

**Review by harshitap26 — APPROVED** [review]

**Review by francis-nijay — APPROVED** [review]

**Pull request merged**